### DISCUSSION TITLE
Is it possible to delete or update a Secret Key?

### DISCUSSION THREAD

**Discussion by ferrata** [darklang/dark#4605] [Q&A] [answered]
Hey there! Playing with the tutorial, I couldn't find a way how I can update or delete a Secret Key. Is it possible?

**Comment by pbiggar** [comment, marked as answer]
It isn't at the moment. The intent is for people to use feature flags to migrate to new secrets, but there's no reason we shouldn't also allow deleting and updating for people who want to do it more directly.

**Reply by pbiggar** [reply]
There's actually an API endpoint enabled to delete secrets, we just didn't add the UI for it yet.

**Reply by ferrata** [reply]
Got it! thanks!

**Reply by StachuDotNet** [reply]
Here's a relevant Issue that tracks against this: https://github.com/darklang/dark/issues/2815
Mentioning here to link the two threads.
